app/models/manageiq/providers/ansible_tower/shared/automation_manager/tower_api.rb

```diff
@@ -8,7 +8,9 @@ def create_in_provider(manager_id, params)
       tower_object = provider_collection(manager).create!(params)
 
       refresh(manager)
-      find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      target = find_by!(:manager_id => manager.id, :manager_ref => tower_object.id)
+      refresh(target)
```

Comment:
Could we do this ^^ for all refresh targets?  It seems weird that a manager refresh isn't a full superset of targeted refresh